### PR TITLE
subject_multiple_rdn.go warning -> notice

### PR DESCRIPTION
**Pull request by zakird**
This addresses #220.

### CODE REVIEW

File: lints/lint_subject_multiple_rdn_test.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 func TestSubjectRDNTwoAttribute(t *testing.T) {
 	inputPath := "../testlint/testCerts/subjectRDNTwoAttribute.pem"
 	expected := Warn
```
Comment (titanous) [inline lints/lint_subject_multiple_rdn_test.go]: Shouldn't this be `Notice` instead of `Warn`? (with the same change to the implementation)

Comment (zakird) [reply lints/lint_subject_multiple_rdn_test.go]: 🤦‍♂️

**Comment by zakird** [comment]
@titanous this should be fixed now. Thanks for catching that. I also updated the lint description.

**Comment by kumarde** [comment]
@zakird I'm not sure if Censys protobufs changed since I last looked at it, but this would likely cause some changes on that end as well. I think it should be as easy as updating the protobuf and running `go generate`, but I would double check that too.

**Pull request merged**